virtcontainers/qemu.go

```diff
@@ -738,9 +738,9 @@ func (q *qemu) startSandbox(timeout int) error {
 		return err
 	}
 	// append logfile only on debug
-	if q.config.Debug {
-		q.qemuConfig.LogFile = filepath.Join(vmPath, "qemu.log")
-	}
+	// if q.config.Debug {
```

Comment:
?

Please either undo the commenting or remove.

Reply:
Yeah i know. That was my dirty fix for #2042. I removed it.